python/fledge/services/core/scheduler/scheduler.py

```diff
@@ -179,6 +179,8 @@ def __init__(self, core_management_host=None, core_management_port=None, is_safe
         """Delete finished task rows when they become this old"""
         self._purge_tasks_task = None  # type: asyncio.Task
         """asynico task for :meth:`purge_tasks`, if scheduled to run"""
+        self._restore_backup_id = None # type: int
+        """Restore backup id and it will be used when SCHEDULE_RESTORE_ON_DEMAND runs"""
```

Comment:
We must avoid such hack! 
I'm failing to understand, why this `_restore_backup_id` is a member of schedule object. We should use a better approach for this and not violate the object oriented design. There can be n number of such type of processes, we should not involve all those in it.